hack/make.sh

```diff
@@ -39,6 +39,14 @@ DEFAULT_BUNDLES=(
 )
 
 VERSION=${VERSION:-dev}
+if [[ $VERSION == refs/tags/* ]]; then
+	VERSION=${VERSION#refs/tags/}
+elif [[ $VERSION == refs/heads/* ]]; then
+	VERSION=$(echo "${VERSION#refs/heads/}" | sed -r 's#/+#-#g')
```

Comment:
You use a here string literally two lines down!
```suggestion
	VERSION=$(<<< "${VERSION#refs/heads/}" sed -r 's#/+#-#g')
```